scss/_buttons.scss

```diff
@@ -38,7 +39,8 @@
   &.zeroclipboard-is-hover,
   &.zeroclipboard-is-active {
     text-decoration: none;
-    @include gradient(#eee, #ddd);
+    background-color: #ddd;
+    background-image: linear-gradient(#eee, #ddd);
```

Comment:
Color literals like `#eee` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.